openlibrary/macros/databarWork.html

```diff
@@ -1,4 +1,12 @@
-$def with (page, editions_page=False, render_times={})
+$def with (page, editions_page=False, render_times={}, summary=None)
+
+$if page.key.startswith('/works'):
+  $ work = page
+$elif page.works:
+  $ work = list(page.works)[0]
+$else:
+  $ work = page.make_work_from_orphaned_edition()
+  $ show_observations = False
```

Comment:
```suggestion
```
This is no longer needed, as the star rating stats are now included in `summary`.